qiskit_nature/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -26,8 +25,6 @@
 from .excited_states_solver import ExcitedStatesSolver
 from .eigensolver_factories import EigensolverFactory
 
-logger = logging.getLogger(__name__)
```

Comment:
I guess you are simply removing these because they are unused, right? I am fine with that for now since we are likely to re-evaluate the logging situation anyways.

Reply:
Yes, the unused loggers have been removed from Terra and I believe Optimization too so I went ahead and removed it in Nature..